api/test/integration/env/configurations/rbac/rule/white_config.txt

```diff
@@ -1,8 +1,8 @@
 | Resource  | List                                                         |
 |-----------|--------------------------------------------------------------|
-| rule:file | 0010-rules_config.xml 0015-ossec_rules.xml local_rules.xml   |
+| rule:file | 0010-rules_config.xml 0015-wazuh_rules.xml local_rules.xml   |
 |           | 0610-win-ms_logs_rules.xml 0620-win-generic_rules.xml ...    |
 
 | Actions    | Resources | Allow                                      | Deny |
 |------------|-----------|--------------------------------------------|------|
-| rules:read | rule:file | 0010-rules_config.xml 0015-ossec_rules.xml |      |
+| rules:read | rule:file | 0010-rules_config.xml                      |      |
```

Comment:
Are we sure about this removal? Shouldn't `0015-wazuh_rules.xml` go here?

Reply:
This change is part of [#6831](https://github.com/wazuh/wazuh/pull/6831) and was validated by the framework team.